Entity/Components/StatusComponent.gd

```diff
@@ -3,38 +3,64 @@ class_name StatusComponent
 ## Applies status to the entity and holds a list of current status.
 
 
-var current_status: Array[StatusBase]
+var current_status: Array[StatusBase] = []
 
+# func _select_storing_array_for_status(status: StatusBase) -> Array[StatusBase]:
```

Comment:
Commented out function here. Do we still need this?

Reply:
not needed anymore, this was when I made two array for the status
I'm removing it